controllers/admin/AdminImportController.php

```diff
@@ -4280,11 +4280,11 @@ protected function excelToCsvFile($filename)
             }
 
             if (!is_file($dest_file)) {
-                $reader_excel = PHPExcel_IOFactory::createReaderForFile($csv_folder . $filename);
+                $reader_excel = \PhpOffice\PhpSpreadsheet\IOFactory::createReaderForFile($csv_folder . $filename);
```

Comment:
Can you please declare the namespace at the top of the file?

```php
use PhpOffice\PhpSpreadsheet\IOFactory;
```

Once done, you can replace this `\PhpOffice\PhpSpreadsheet\IOFactory` with `IOFactory`.

Reply:
```suggestion
                $reader_excel = IOFactory::createReaderForFile($csv_folder . $filename);
```

Reply:
Thanks for the suggestion, that's a neater way of doing it.